src/components/templates/OcTableFiles/OcTableFiles.vue

```diff
@@ -13,8 +13,8 @@
     @highlight="fileClicked"
     @rowMounted="rowMounted"
     @contextmenuClicked="showContextMenu"
-    @fileDropped="fileDropped"
-    @fileDragged="fileDragged"
+    @itemDropped="fileDropped"
```

Comment:
This was an unnecessary breaking change in retrospect (it's the OcFilesTable so file as a prefix is fine, slipped through my code review), let's discuss if we want to rollback & re-release or ship a bugfix release tomorrow